server/src/main/java/com/github/phantazmnetwork/server/ZombiesTest.java

```diff
@@ -57,5 +66,17 @@ static void initialize(@NotNull EventNode<Event> global) {
                 everyoneElseHologram.setInstance(spawnInstance);
             }
         });
+
+        global.addListener(EventListener.builder(PlayerChatEvent.class).ignoreCancelled(false).handler(event -> {
+            if (event.getMessage().equals("G")) {
+                event.getPlayer().openInventory(Gui.builder(InventoryType.CHEST_6_ROW, new BasicSlotDistributor(1))
+                        .withItem(GuiItem.builder().withItem(ItemStack.of(Material.SEAGRASS)
+                                        .withDisplayName(Component.text("Become beaned").color(TextColor.color(0, 255, 0))))
```

Comment:
this is test code which you could also just remove entirely but Component.text(String, TextColor) exists